src/etc/inc/vpn.inc

```diff
@@ -448,4 +426,33 @@ EOD;
 	return 0;
 }
 
+function vpn_l2tp_updatesecret() {
+	global $config, $g;
+	$l2tpcfg = $config['l2tp'];
+
+	$fd = fopen("{$g['varetc_path']}/l2tp-vpn/mpd.secret", "w");
+	if (!$fd) {
+		printf(gettext("Error: cannot open mpd.secret in vpn_l2tp_configure().") . "\n");
```

Comment:
This description isn't accurate anymore (the function name changed).

Reply:
ok, fixed